meta-facebook/yv35-cl/src/platform/plat_ipmb.c

```diff
@@ -0,0 +1,59 @@
+#include <stdio.h>
+#include "cmsis_os2.h"
+#include <string.h>
+#include "plat_i2c.h"
+#include "plat_ipmb.h"
+#include "plat_ipmi.h"
+#include "plat_class.h"
+
+IPMB_config pal_IPMB_config_table[] = {
+	// index, interface, channel, bus, channel_target_address, enable_status, self_address,
+	// rx_thread_name, tx_thread_name
+	{ BMC_IPMB_IDX, I2C_IF, BMC_IPMB, IPMB_I2C_BMC, BMC_I2C_ADDRESS, ENABLE, SELF_I2C_ADDRESS,
+	  "RX_BMC_IPMB_TASK", "TX_BMC_IPMB_TASK" },
+	{ ME_IPMB_IDX, I2C_IF, ME_IPMB, IPMB_ME_BUS, ME_I2C_ADDRESS, ENABLE, SELF_I2C_ADDRESS,
+	  "RX_ME_IPMB_TASK", "TX_ME_IPMB_TASK" },
+	{ EXP1_IPMB_IDX, I2C_IF, EXP1_IPMB, IPMB_EXP1_BUS, BIC1_I2C_ADDRESS, DISABLE,
+	  SELF_I2C_ADDRESS, "RX_EPX1_IPMB_TASK", "TX_EXP1_IPMB_TASK" },
+	{ EXP2_IPMB_IDX, I2C_IF, EXP2_IPMB, IPMB_EXP2_BUS, BIC2_I2C_ADDRESS, DISABLE,
+	  SELF_I2C_ADDRESS, "RX_EPX2_IPMB_TASK", "TX_EXP2_IPMB_TASK" },
+	{ RESERVED_IDX, RESERVED_IF, RESERVED, RESERVED_BUS, RESERVED_ADDRESS, DISABLE,
+	  RESERVED_ADDRESS, "RESERVED_ATTR", "RESERVED_ATTR" },
+};
+
+bool pal_load_ipmb_config(void)
```

Comment:
This function always returns true. Should it be a void return type instead?

Reply:
The return code is checked in ipmb_init() function.
We have fixed it and done the forced update.